### DISCUSSION TITLE
HomeAssistant Add-on und HmIPW-DRAP

### DISCUSSION THREAD

**Discussion by oliverboermel** [OpenCCU/OpenCCU#2109] [🇩🇪 Fragen&Antworten] [answered]
Ich bekomme den Drap in meiner Raspberrymatic HA Addon  nicht angemeldet
was mach ich falsch Firmware ist aktuell

```
Mounting /data as /usr/local (Home Assistant Add-On): OK
Starting watchdog...
Identifying host system: oci, OK
Initializing RTC Clock: no hardware found
Running sysctl: OK
Checking for Factory Reset: not required
Checking for Backup Restore: not required
Initializing System: OK
Setup ca-certificates: OK
Starting logging: OK
Init onboard LEDs: init, OK
Starting irqbalance: OK
Starting iptables: OK
Starting network: eth0: link up, fixed, firewall, inet up, 172.30.33.3, OK
Identifying Homematic RF-Hardware: ....HmRF: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.1.1, HmIP: HMIP-RFUSB/eQ-3 HmIP-RFUSB@usb-0000:01:00.0-1.1.1, OK
Updating Homematic RF-Hardware: HMIP-RFUSB: 4.4.18, not necessary, OK
Starting hs485dLoader: disabled
Starting xinetd: OK
Starting eq3configd: OK
Starting lighttpd: OK
Starting ser2net: disabled
Starting ssdpd: OK
Starting ha-proxy: OK
Starting NUT services: disabled
Initializing Third-Party Addons: OK
Starting LGWFirmwareUpdate: ...OK
Setting LAN Gateway keys: OK
Starting hs485d: disabled
Starting multimacd: .......OK
Starting rfd: .OK
Starting HMIPServer: .........................OK
Starting ReGaHss: .OK
Starting CloudMatic: OK
Starting NeoServer: disabled
Starting Third-Party Addons: OK
Starting crond: OK
Setup onboard LEDs: booted, OK
Finished Boot: 3.65.11.20221218 (raspmatic_oci_arm64)
```

**Comment by Baxxy13** [comment, marked as answer]
> was mach ich falsch

Du hast die bekannten Limitierungen des Homeassistant-AddOns überlesen oder ignoriert. 

DRAP und auch HAP (als LAN-Router) lassen sich nicht mit dem AddOn nutzen.

**Reply by jens-maus** [reply]
Und um das noch einmal klarer zu machen, hier der Link zu den "Known Limitations" die hier wohl überlesen wurden:

https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant#known-limitations

**Comment by oliverboermel** [comment]
ja das hab ich wohl überlesen sorry. wird sich das noch ändern? sonst muss ich wohl alles umbauen.

**Reply by jens-maus** [reply]
Irgendwann wird sich das vmtl. mal ändern, ja. Aber nicht auf absehbar kurze Zeit. Wir reden hier von mehreren Monaten und das Feature hat bei mir selbst eine gewisse niedrige Priorität, ehrlich gesagt.

**Reply by oliverboermel** [reply]
ok gut zu wissen auch wenn es schade ist da ich gerade alles auf wired umbaue aber dann muss ich wohl nen HAP nehmen 
danke für die Auskunft und schöne Weihnachtstage
